js/private/test/js_run_devserver/js_run_devserver_test.bzl

```diff
@@ -0,0 +1,24 @@
+load("//js/private:js_run_devserver.bzl", "js_run_devserver", "js_run_devserver_lib")
+
+_js_run_devserver_test = rule(
+    attrs = js_run_devserver_lib.attrs,
+    implementation = js_run_devserver_lib.implementation,
+    toolchains = js_run_devserver_lib.toolchains,
+    test = True,
+)
+
+# 'test' version of js_run_devserver
+def js_run_devserver_test(
+        name,
+        tags = [],
+        **kwargs):
+
+    js_run_devserver(
+        name,
+        js_run_devserver_rule = _js_run_devserver_test,
```

Comment:
Here's where I'm using the new attribute of `js_run_devserver`. Again, let me know if there's a better way to have that macro use the "test" version of the rule.